homeassistant/bootstrap.py

```diff
@@ -407,6 +407,8 @@ def _init_blocking_io_modules_in_executor() -> None:
     # Initialize the mimetypes module to avoid blocking calls
     # to the filesystem to load the mime.types file.
     mimetypes.init()
+    is_official_image()
```

Comment:
Should comments be added?

Reply:
I figured the name of the function said it all, but I guess not 😉 
Added a comment

Reply:
Thanks